extensions/wikia/ARecoveryEngine/js/PageFair/pageFair.js

```diff
@@ -19,19 +20,27 @@ define('ext.wikia.aRecoveryEngine.recovery.pageFair', [
 	}
 
 	function isSlotRecoverable(slotName) {
-		var result = isPageFairRecoveryEnabled() && !!recoverableSlots[slotName];
+		var result = isPageFairRecoveryEnabled() && recoverableSlots.indexOf(slotName) !== -1;
 
 		log(['isSlotRecoverable', result], log.levels.info, logGroup);
 		return result;
 	}
 
+	function isBlocking() {
+		var isBlocking = !!(win.ads && win.ads.runtime.pf && win.ads.runtime.pf.blocking);
+
+		log(['isBlocking', isBlocking], 'debug', logGroup);
```

Comment:
We can also replace `'debug'` in all places with `log.levels.debug` as :cherries: ;-)